src/platform/networking/common/networking.ts

```diff
@@ -151,6 +151,11 @@ export interface ICreateEndpointBodyOptions extends IMakeChatRequestOptions {
 	postOptions: OptionalChatRequestParams;
 }
 
+export interface ICustomModel {
```

Comment:
Lets just have whatever interface or type we use be the same as the ModelAPIResponse. Look at `ModelPolicy` for an example. 

This way we dont bother with maintaining two types